### PR TITLE
Feat(test): add comparison test

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:

Feat(test): add comparison test

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @simon-cj @gaocegege 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/registry/harbor/harbor.go [hunk 1]
```diff
@@ -16,22 +16,28 @@ const (
 )
 
 // Proxy is the proxy to Harbor core service.
-type Proxy struct {
+type Proxy interface {
```
Comment (simon-cj) [inline pkg/registry/harbor/harbor.go:19]: Proxy and proxy is confuse， can rename it

**Review by gaocegege — COMMENTED** [review]
/approve

**Pull request merged**